faker/utils/distribution.py

```diff
@@ -17,10 +18,13 @@ def cumsum(it):
 def choice_distribution(a, p):
     assert len(a) == len(p)
 
-    cdf = list(cumsum(p))
-    normal = cdf[-1]
-    cdf2 = [float(i) / float(normal) for i in cdf]
-    uniform_sample = random_sample()
-    idx = bisect.bisect_right(cdf2, uniform_sample)
-
-    return a[idx]
+    if version_info.major >= 3 and version_info.minor >= 6:
```

Comment:
Good call! Looks like hasattr is quite a bit faster, at least when testing on python 3.6 on my machine.

```
$ python3 -m timeit "import random; hasattr(random, 'choices')"
1000000 loops, best of 3: 0.265 usec per loop
$ python3 -m timeit "import sys; sys.version_info.major >= 3 and sys.version_info.minor >= 6"
1000000 loops, best of 3: 0.614 usec per loop
```

I'll update the PR soon. :)